lona/html/nodes.py

```diff
@@ -184,17 +184,19 @@ class Img(Node):
 class A(Node):
     TAG_NAME = 'a'
 
-    def __init__(self, *args, interactive=False, **kwargs):
+    def __init__(self, *args, interactive=None, **kwargs):
```

Comment:
@maratori No, i am afraid not. All Lona nodes have a property ``ignore`` which tells the javascript client to not patch the default behavior of it. ``interactive`` is a helper property that implements the same wording like in https://lona-web.org/end-user-documentation/views.html#view-types . ``interactive`` uses ``ignore`` internally. If ``interactive`` is always set, ``ignore`` does not work as expected.

Reply:
`node.ignore` is `False` for all nodes (including `A`) by default. `A().interactive` is `True` by default. So why it will not work if default constructor argument will be `True`?

Reply:
If you set ``ignore=True`` it gets overridden implicitly by if ``interactive`` has the default ``True``, which is confusing

Reply:
Ah. Now I see. I think it's better to add this comment to code.

Reply:
I agree. Fixed